.eslintrc

```diff
@@ -4,33 +4,38 @@
     "mozilla",
     "react"
   ],
+  "extends": [
+    "eslint:recommended"
+  ],
   "rules": {
     "mozilla/components-imports": 1,
     "mozilla/import-globals-from": 1,
     "mozilla/this-top-level-scope": 1,
-    "no-unused-vars": 2,
-    "semi": [2, "always"],
-    "no-undef": 2,
-    "object-curly-spacing": [2, "never"],
-    "computed-property-spacing": [2, "never"],
-    "array-bracket-spacing": [2, "never"],
-    "space-before-function-paren": [2, {"anonymous": "never", "named": "never"}],
+
     "react/jsx-uses-react": 1,
+
+    "array-bracket-spacing": [2, "never"],
+    "comma-dangle": 0,
+    "computed-property-spacing": [2, "never"],
+    "no-console": 1,
```

Comment:
All of this green-vs-red mess was just me A-Zing the rules, although I added `no-console: 1` (since I believe "eslint:recommended" sets it to an error by default).